### ISSUE TITLE
magic.MagicException: line 60: regex error 16

### ISSUE THREAD

**Issue by cr** [ahupp/python-magic#72]
I'm running python-magic==0.4.6 on a large dataset with ```mime=True```, and one .html file triggered the following MagicException:
```
  File "/.../lib/python2.7/site-packages/magic.py", line 66, in from_buffer
    return magic_buffer(self.cookie, buf)
  File "/.../lib/python2.7/site-packages/magic.py", line 215, in magic_buffer
    return _magic_buffer(cookie, buf, len(buf))
  File "/.../lib/python2.7/site-packages/magic.py", line 168, in errorcheck_null
    raise MagicException(err)
magic.MagicException: line 60: regex error 16 for `^\s*except.*:', (invalid argument to regex routine)

```

**Comment by cr** [comment]
Correction, it's a JavaScript text file which file-5.04 on OS X reports as "ASCII C++ program text".

**Comment by ahupp** [comment]
This looks like an issue in your underlying libmagic implementation, which python-magic uses.  I'd suggested debugging there, and/or reporting to that team.  Sorry I can't be more help.

**Comment by pombreda** [comment]
@cr this is likely fixed in 5.23 released a few days ago: Per http://mx.gw.com/pipermail/file/2015/001739.html
I am saying so because I did hit the same bug in 5.22 
"* Fix issue with regex range for magic with offset"
